.github/workflows/python-package.yml

```diff
@@ -49,6 +49,10 @@ jobs:
       run: |
         source ./venv/bin/activate
         make lint
+    - name: Test
```

Comment:
This step has the same name as the one in line 56. Different names would be more helpful.